index.html

```diff
@@ -112,6 +112,33 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ================================================ -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- Pankaj Sharma Card -->
+
+        <div class="card">
+          <p class="name">Pankaj Sharma</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/Pankaj_sharma43" target="_blank">@Pankaj_sharma43</a>
+          </p>
+          <p class="about">I am a Fullstack developer | JS, NodeJS, React, MongoDB, SQL | Code is Fun | Experience in building real-time web apps.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+	           <ul>
+              <li>
+                <a href="https://javascript.info/" target="_blank" title="Javascript info">Javascript Info</a>
+              </li>
+              <li>
+                <a href="https://dev.to/" target="_blank" title="Second Resource">Dev.to</a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Third resource">StackOverflow</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
```

Comment:
Remove this blank space also before your "card end comment"

Reply:
Code is updated. Plz take a look